web/adventures/models.py

```diff
@@ -0,0 +1,51 @@
+from django.db import models
+
+def parse_URL_list_field(url_list, separator):
+    """Takes a string of urls and coverts them into a list."""
+    if url_list == "":
+        return []
+    return url_list.split(",")
+
+class URLListField(models.URLField):
```

Comment:
Hmm, I was thinking that too, but I'm not sure that would make sense. It's essentially stored as a text field, but I expect people to interact with it as a list of strings. I wouldn't say that a list of strings is a subtype of a string, so does it make sense to say a list of URLs is a subtype of `CharField`?